tasks/config/behatConfig.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function (grunt) {
```

Comment:
If the name of the grunt module is "behat" this file should probably be named "behat"